src/main/java/iudx/file/server/apiserver/FileServerVerticle.java

```diff
@@ -641,7 +643,7 @@ public void delete(RoutingContext routingContext) {
           String resultTitle = dbHandlerResult.getJsonObject("result").getString("title");
           int resultType = dbHandlerResult.getJsonObject("result").getInteger("type");
           HttpStatusCode code = HttpStatusCode.getByValue(resultType);
-          ResponseUrn urn = ResponseUrn.fromCode(resultTitle);
+          ResponseUrn urn = fromCode(resultTitle);
```

Comment:
```suggestion
          ResponseUrn urn = urnFromCode(resultTitle);
```
better to change the method name.